Deprecation warnings

```
In file included from /app/webkit/Tools/wpe/cog/core/cog-launcher.c:9:
/app/webkit/Tools/wpe/cog/core/cog-launcher.h:22:35: warning: redefinition of typedef 'CogRequestHandler' is a C11 feature [-Wtypedef-redefinition]
typedef struct _CogRequestHandler CogRequestHandler;
                                  ^
/app/webkit/Tools/wpe/cog/core/cog-request-handler.h:21:22: note: previous definition is here
G_DECLARE_INTERFACE (CogRequestHandler, cog_request_handler, COG, REQUEST_HANDLER, GObject)
                     ^
1 warning generated.
[8/16] Building C object CMakeFiles/cog.dir/cog.c.o
In file included from /app/webkit/Tools/wpe/cog/cog.c:13:
In file included from /app/webkit/Tools/wpe/cog/core/cog.h:19:
/app/webkit/Tools/wpe/cog/core/cog-launcher.h:22:35: warning: redefinition of typedef 'CogRequestHandler' is a C11 feature [-Wtypedef-redefinition]
typedef struct _CogRequestHandler CogRequestHandler;
                                  ^
/app/webkit/Tools/wpe/cog/core/cog-request-handler.h:21:22: note: previous definition is here
G_DECLARE_INTERFACE (CogRequestHandler, cog_request_handler, COG, REQUEST_HANDLER, GObject)
                     ^
/app/webkit/Tools/wpe/cog/cog.c:283:5: warning: 'webkit_web_context_set_tls_errors_policy' is deprecated: Use 'webkit_website_data_manager_set_tls_errors_policy' instead [-Wdeprecated-declarations]
    webkit_web_context_set_tls_errors_policy (cog_shell_get_web_context (shell),
    ^
/app/webkit/WebKitBuild/Release/DerivedSources/ForwardingHeaders/wpe/wpe/WebKitWebContext.h:250:1: note: 'webkit_web_context_set_tls_errors_policy' has been explicitly marked deprecated here
WEBKIT_DEPRECATED_FOR(webkit_website_data_manager_set_tls_errors_policy) void
^
/app/webkit/WebKitBuild/Release/DerivedSources/ForwardingHeaders/wpe/wpe/WebKitDefines.h:46:45: note: expanded from macro 'WEBKIT_DEPRECATED_FOR'
#define WEBKIT_DEPRECATED_FOR(f) WEBKIT_API G_DEPRECATED_FOR(f)
                                            ^
/usr/include/glib-2.0/glib/gmacros.h:952:44: note: expanded from macro 'G_DEPRECATED_FOR'
#define G_DEPRECATED_FOR(f) __attribute__((__deprecated__("Use '" #f "' instead")))
                                           ^
2 warnings generated.

```

--- a/core/cog-launcher.h
+++ b/core/cog-launcher.h
@@ -14,19 +14,16 @@
 #endif
 
 #include "cog-config.h"
+#include "cog-request-handler.h"
 #include "cog-shell.h"
-#include "cog-webkit-utils.h"
 
 G_BEGIN_DECLS
 
-typedef struct _CogRequestHandler CogRequestHandler;
+#define COG_TYPE_LAUNCHER (cog_launcher_get_type())
 
-#define COG_TYPE_LAUNCHER (cog_launcher_get_type ())
+G_DECLARE_FINAL_TYPE(CogLauncher, cog_launcher, COG, LAUNCHER, GApplication)
 
-G_DECLARE_FINAL_TYPE (CogLauncher, cog_launcher, COG, LAUNCHER, GApplication)
-
-struct _CogLauncherClass
-{
+struct _CogLauncherClass {
     GApplicationClass parent_class;
 };
 

